our-team/guide-our-team.md

```diff
@@ -0,0 +1,159 @@
+# Our Team template guide
+
+---
+Thank you for downloading the Our Team template from The Good Docs Project!
+We welcome any kind of feedback on our templates, both positive and negative.
+To suggest improvements, open an issue on our (https://github.com/thegooddocsproject/templates/issues)[Templates repository].
+---
+
+The `Our Team` template is for open source software projects.
+The template helps you clearly communicate who belongs to your open source project or organization.
+This template:
+
+- Lists the teams or roles for your organization or project.
+- Explains the purpose of each team or role.
+- Provides the names of team members in these roles and, optionally, the contact information for each person.
+
+Think of this template as the start of your document.
+You can feel free to disregard some of the suggestions made in this template and write sections with your own unique content if your document feels like it needs something different.
+
+
+## Why do I need the Our Team template?
+At the end of the day, open source communities are powered by people.
+The success of an open source project depends less on the cool tech or resources you're building and more about the health of the community of people who both contribute to the project and use the technologies you create.
+To help your project succeed, both your users and your contributors need to be able to communicate effectively with you to accomplish important tasks, including (but not limited to):
+
+- Joining your community and collaborating with you to exchange ideas and help your technology improve.
+- Making your technology better by letting you know about bugs or usability improvements that can help increase the chances your technology will be adopted.
+- Evangelizing your project by inviting you to speak at events or conferences or sponsoring meetups to learn more about your technology.
+- Supporting your project financially.
+
+To make it easier for people to accomplish these tasks, it helps tremendously to have a page in your repository or website that explains who are in key roles and teams in your project.
+
+Additionally, creating this content will help you and your project get more clear about what roles you have and what they mean.
+The `Our Team` template will help you think through these roles and create this content.
+
+
+## What kind of teams and roles should my project have?
+
+The teams and roles your project needs will depend on the type of organization you are, how mature your project is, what your community is like, and your community governance style.
+In other words, it depends on a lot of different factors.
+
+For now, just focus on describing your teams and roles the way they currently exist in your project.
+
+However, if you want to think more deeply about effective open source governance, consider checking out these resources:
+
+- (https://opensource.guide/leadership-and-governance/)[Leadership and governance] (opensource.guide)

```

Comment:
todo (blocking): Link syntax should be `[Leadership and goverance](https://opensource.guide/leadership-and-governance/)"`